boards/arm/frdm_k82f/frdm_k82f_defconfig

```diff
@@ -11,6 +11,7 @@ CONFIG_PINMUX=y
 CONFIG_GPIO=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=120000000
 CONFIG_OSC_LOW_POWER=y
+CONFIG_COUNTER=y
```

Comment:
This will enable counter for all applications, even hello_world and blinky, which we don't want to do

Reply:
right, I forget to change this, nice catch.